feature-libs/checkout/scheduled-replenishment/components/services/checkout-replenishment-form-service.ts

```diff
@@ -25,17 +44,74 @@ export class CheckoutReplenishmentFormService {
   private scheduleReplenishmentFormData$: BehaviorSubject<ScheduleReplenishmentForm> =
     new BehaviorSubject<ScheduleReplenishmentForm>(this.defaultFormData);
 
-  constructor() {}
+  protected orderType$ = new BehaviorSubject<ORDER_TYPE>(
```

Comment:
In order to prevent using the `orderType$` directly, let's type it as
```suggestion
  protected orderType$: Observable<ORDER_TYPE> = new BehaviorSubject<ORDER_TYPE>(
    ORDER_TYPE.PLACE_ORDER
  );
```